File: WillShop/app/src/main/java/com/will/willshop/MainActivity.java
package com.will.willshop;

import android.os.Bundle;

import com.will.common.ui.component.WiBaseActivity;
import com.will.ui.tab.bottom.WiTabBottomLayout;
import com.will.willshop.logic.MainActivityLogic;

public class MainActivity extends WiBaseActivity implements MainActivityLogic.ActivityProvider {
    MainActivityLogic mActivityLogic;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mActivityLogic = new MainActivityLogic(this);
    }
}

File: hiui/app/src/main/java/com/will/hi_ui/MainActivity.kt
package com.will.hi_ui

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.will.hi_ui.tab.bottom.HiTabBottom
import com.will.hi_ui.tab.bottom.HiTabBottomInfo

class MainActivity : AppCompatActivity(), View.OnClickListener {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val tabBottom = findViewById<HiTabBottom>(R.id.tab_bottom)

        val homeInfo = HiTabBottomInfo(
            "首页",
            "fonts/iconfont.ttf",
            getString(R.string.if_home),
            null,
            "#ff656667",
            "#ffd44949"
        )
        tabBottom.setHiTabInfo(homeInfo)
    }

    override fun onClick(v: View?) {

    }
}
